framework/src/main/java/framework/java/measurement/MeasurementUtils.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * The MIT License (MIT)
+ * Copyright (c) 2016 FI MUNI
+ */
+package main.java.framework.java.measurement;
+
+import java.nio.file.Path;
+
+import org.apache.commons.lang.reflect.FieldUtils;
+import org.sonar.plugins.java.api.JavaFileScannerContext;
+import org.sonar.plugins.java.api.tree.ClassTree;
+import org.sonar.plugins.java.api.tree.IdentifierTree;
+import org.sonar.plugins.java.api.tree.MemberSelectExpressionTree;
+import org.sonar.plugins.java.api.tree.ParameterizedTypeTree;
+import org.sonar.plugins.java.api.tree.TypeTree;
+
+/**
+ * Utils class for java measurements
+ * @author Tomas
+ */
+public class MeasurementUtils {
+
+	/**
+	 * @param tree
+	 * @return
+	 */
+	static String extractTreeSimpleName(TypeTree tree) {
```

Comment:
to je package viditelnost, to nema zmysel pouzivat inde ... mozno to na reflexiu ano